packages/configs/README.md

```diff
@@ -92,37 +92,37 @@ To use the `eslint` configs in an Amplify UI package, declare `eslint-config-amp
 
 > Note the dependency version of `0.0.0`. Rather than publishing our custom `eslint` configs to NPM, we utilize the `workspaces` field of the top level [_package.json_](../../package.json) to allow our packages to use the configs during development.
 
-**`eslint-config-amplify-ui`/`eslint-config-amplify-ui/react`**
+**`@aws-amplify/eslint-config-amplify-ui`/`@aws-amplify/eslint-config-amplify-ui/react`**
 
-Add the desired config to the `extends` field of the desired package _.eslintrc.js_, along with the noted `parserOptions`. The configs can be extended using either short (`'amplify-ui'`) or long (`'eslint-config-amplify-ui'`) form syntax:
```

Comment:
I didn't find where we set the short form and it no longer works, so I deleted it to make it easier to find for the similar scenarios in the future.

Reply:
fyi, shorthands are eslint feature: https://eslint.org/docs/latest/use/configure/configuration-files#extending-configuration-files

Reply:
Thanks! I see - We can "omit the prefix". I'll keep the long version so that it'll be less likely to miss it when doing a global search.

Reply:
Not sure why we are removing the short form syntax?

Reply:
Yeah, let's add them back in if shorthand syntax works with scoped packages.

Reply:
The short hand is easy to be missed. What's the benefit of using it?